mzLib/MzLibUtil/MrsNoiseEstimation/MRSNoiseEstimator.cs

```diff
@@ -47,8 +45,11 @@ public static class MRSNoiseEstimator
                 // 4. Compute the multiresolution support M that is derived from the wavelet coefficients
                 // and the standard deviation of the noise at each level. 
                 // 5. Select all points that belong to the noise; they don't have an significant coefficients above noise 
+                // 1.97 is the z-score representing a value that is two standard deviations away from the mean. 
```

Comment:
This is incorrect. Z-score represents the number of standard deviations separating a measurement from the mean. Measurments with a z-score < 1.97 represent 97.6% of a normal distribution.